test_path.py

```diff
@@ -312,15 +320,17 @@ def test_read_write(self, tmpdir):
 
 class TestPerformance:
     @staticmethod
-    def get_command_time(cmd):
+    def get_command_time(cmd: str) -> datetime.timedelta:
         args = [sys.executable, '-m', 'timeit', '-n', '1', '-r', '1', '-u', 'usec'] + [
             cmd
         ]
         res = subprocess.check_output(args, universal_newlines=True)
-        dur = re.search(r'(\d+) usec per loop', res).group(1)
+        m = re.search(r'(\d+) usec per loop', res)
+        assert m is not None
+        dur = m.group(1)
```

Comment:
I prefer the previous implementation. Maybe it would make sense to just ignore type checks for tests.

Reply:
Of course, that's up to you. If not run on the tests, one might add a type annotation that is not accurate but not know because the annotation is not checked against "real code". 

As I mentioned above, ideally, one could tell `mypy` to typecheck the tests without having to add all redundant annotations on the checks and backwards-bending for common regex idioms or platform-dependent code, but alas.